project/detector.py

```diff
@@ -0,0 +1,44 @@
+"""
+Sentiment detector for Tidbits:

```

Comment:
nit: put the summary on the same line as the initial """

Reply:
"""Sentiment detect for Tidbits.

Utilizes...
"""

Reply:
done